src/Motor.Extensions.Hosting.RabbitMQ/Options/RabbitMQBaseOptions.cs

```diff
@@ -1,22 +1,36 @@
 using System;
+using System.Net.Security;
+using System.Security.Authentication;
 using Motor.Extensions.Hosting.RabbitMQ.Validation;
 
 namespace Motor.Extensions.Hosting.RabbitMQ.Options;
 
+public sealed record RabbitMQSslOptions
+{
+    public bool Enabled { get; init; } = false;
+    public SslProtocols Protocols { get; init; } = SslProtocols.Tls12;
+
+    public string ClientCertPath { get; init; } = string.Empty;
```

Comment:
(nitpick) I think an XML doc comment would be good for this and the next option that also mentions that client certificates are optional.